### DISCUSSION TITLE
Unable to start interpreter on remote

### DISCUSSION THREAD

**Discussion by clydeugene** [posit-dev/positron#4715] [Q&A] [answered]
Hello everyone,

I just discovered that remote connections are now available on Positron 🎉. However, I'm unable to start any interpreter when connected to a remote session. When I click on the `Start Interpreter` button, it's as if the pop-up is hiding behind some other UI element. If anyone has had this issue, please may you help me figure it out.

I am running Positron 2024.09.0 (Universal) build 45 on an M1 MacBook Air, while connecting to an Ubuntu 20.04.5 LTS server. I have R 4.3.1 installed on the server as well as Python 3.12.6 - all as an unprivileged user.

<img width="2304" alt="image" src="https://github.com/user-attachments/assets/0a40068f-6b5f-47d3-b72d-3c3e9e93eaad">

I thought to post this here before creating an issue.

Clyde

**Comment by jmcphers** [comment, marked as answer]
Thanks for trying the experimental SSH feature! Unfortunately, Positron currently has some known issues on Ubuntu 20 that lead to the behavior you're seeing. We're looking into them. You can follow the discussion here: https://github.com/posit-dev/positron/issues/3854. Are you able to try with Ubuntu 22 or 24?

Separately, we also have an issue tracking showing better UX when Positron can't find/connect to any interpreters. https://github.com/posit-dev/positron/issues/3468

**Reply by clydeugene** [reply]
Thanks @jmcphers, I followed the discussion on #3854 and my mind was blown away by the number of things that you have to deal with to give us such a useful product  🤯 

Keep up the good work! I've just subscribed for updates.

**Reply by pwilnerjeanty** [reply]
I am using Red Hat Enterprise Linux release 8.10 and dealing with the same issue. When clicking on the drop-down menu, I only see Start Interpreter underlined with a bold line.

Thank you for all the work you have put into this.

**Reply by jmcphers** [reply]
Yes, RHEL 8.1 also has a very old glibc (2.28, [source](https://distrowatch.com/table.php?distribution=redhat)). We're working on it.
